### DISCUSSION TITLE
Monitored sites name

### DISCUSSION THREAD

**Discussion by fez7yoe** [upptime/upptime#629] [Q&A]
Entering a website name in a language other than english (e.g. Chinese) may result in incomplete display of information on that monitored website.
for example, history, graphics, api and other information.
even if the information is generated, the file name generated by the history is <code>.yml</code>
Yes, the .yml file without a name

**Comment by AnandChowdhary** [comment]
Aah that's a bummer :( This is likely because the `slugify` function that converts the site name to file name (e.g., "English Wikipedia" becomes "english-wikipedia") doesn't support non-Latin characters. A good solution may be to somehow transliterate the site name or perhaps an option to use the user input directly?

**Reply by fez7yoe** [reply]
The use of english will not have this problem
yes, that's what you mean by phonetic or related english names
but you said the option to use the user input directly is?
i don't understand this part about user input
